### DISCUSSION TITLE
API 란 무엇인가?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#8] [Q&A] [answered]
API 란 무엇인가?

**Comment by tjdud22** [comment, marked as answer]
운영체제와 응용프로그램 사이의 통신에 사용되는 언어나 메시지 형식을 이야기합니다.
API는 중간에서 양쪽의 서버를 연결을 해줍니다.

**Comment by tjdud22** [comment]
여러 프로그램들과 데이터베이스, 그리고 기능들의 상호 통신 방법을 규정하고 도와주는 매개체입니다.

**Comment by tjdud22** [comment]
“Application Programming Interface”를 말한다.
둘이상의 컴퓨터 프로그램이 서로 통신하는 방법이자
컴퓨터 사이에 있는 중개계층을 의미한다.
예를들어 프로토콜,메서드,데이터타입등이 정의된 중개계층입니다
